src/main/java/xyz/deseteral/lebkuchenfm/services/CommandParser.java

```diff
@@ -0,0 +1,30 @@
+package xyz.deseteral.lebkuchenfm.services;
+
+import org.springframework.stereotype.Component;
+import xyz.deseteral.lebkuchenfm.domain.Command;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Component
+public class CommandParser {
+    public Optional<Command> parse(String text) {
```

Comment:
Może by tu zwracać po prostu Command, a w razie jak coś pójdzie nie tak to rzucać wyjątkiem? Zwracanie optionali z serwisów domenowych ma swoje wady i zalety :)

Reply:
Rzucam już innym wyjątkiem w `CommandExecutor`, więc wydawało mi się że lepiej trzymać to w jednym miejscu